server/src/test/java/org/elasticsearch/cluster/ClusterInfoTests.java

```diff
@@ -44,12 +44,13 @@ private static Map<String, DiskUsage> randomDiskUsage() {
         Map<String, DiskUsage> builder = new HashMap<>(numEntries);
         for (int i = 0; i < numEntries; i++) {
             String key = randomAlphaOfLength(32);
+            final int totalBytes = randomIntBetween(0, Integer.MAX_VALUE);
```

Comment:
Had to fix this to use plausible values, otherwise `toString` would run into negative byte values which would throw illegal argument ex.